### PR TITLE
[ko] Fix kube-controller-manager information in components.md

### PR DESCRIPTION
**Pull request by copyx**
I change the description about replication controller(레플리케이션 컨트롤러) to job controller.

[In english version of same page](https://kubernetes.io/docs/concepts/overview/components/#kube-controller-manager), there is no description about replication controller.

### CODE REVIEW

**Comment by sftim** [comment]
The localized page is stale: the English page previously did mention ReplicationController.

**Comment by seokho-son** [comment]
/retitle [ko] Fix kube-controller-manager information in components.md

File: content/ko/docs/concepts/overview/components.md [hunk 1]
```diff
@@ -51,8 +51,7 @@ card:
 이들 컨트롤러는 다음을 포함한다.
 
   * 노드 컨트롤러: 노드가 다운되었을 때 통지와 대응에 관한 책임을 가진다.
-  * 레플리케이션 컨트롤러: 시스템의 모든 레플리케이션 컨트롤러 오브젝트에 대해 알맞은 수의 파드들을
-  유지시켜 주는 책임을 가진다.
+  * 잡 컨트롤러: 일회성 작업을 나타내는 잡 오브젝트를 감시한 다음, 해당 작업을 완료할 때까지 실행하는 파드를 생성한다.
```
Comment (seokho-son) [inline content/ko/docs/concepts/overview/components.md:54]:
```suggestion
  * 잡 컨트롤러: 일회성 작업을 나타내는 잡 오브젝트를 감시한 다음, 해당 작업을
    완료할 때까지 동작하는 파드를 생성한다.
```
한글 문서의 개행은 코드 유지보수를 위해서 원문과 유사하게 (가급적 동일하게)
처리하고 있습니다. 개행을 부탁드립니다. :)

추가적으로 실행하는 -> 동작하는 으로 자연스럽게 변경해보았습니다. :)

```
Line 54:  * Job controller: Watches for Job objects that represent one-off tasks, then creates
Line 55:    Pods to run those tasks to completion.
```

**Review by seokho-son — COMMENTED** [review]
Hi, @copyx 
Thank you for the contribution !!
I have minor comments below,

**Comment by seokho-son** [comment]
> The localized page is stale: the English page previously did mention ReplicationController.

Hi @sftim ,

I've checked https://kubernetes.io/docs/concepts/overview/components and https://kubernetes.io/ko/docs/concepts/overview/components/

The localized page has a few outdated contents. 
Those outdated contents can be updated by the task M18 in #36863 and this PR.

File: content/ko/docs/concepts/overview/components.md (hunk 1 above)
Comment (seokho-son) [reply content/ko/docs/concepts/overview/components.md:54]: Please squash commits (or amend) when you update this PR. :)

Comment (seokho-son) [reply content/ko/docs/concepts/overview/components.md:54]: PTAL @copyx  :)

I think you can update this PR easily when you notice this message :)

Comment (copyx) [reply content/ko/docs/concepts/overview/components.md]: 피드백 주신 내용 적용 완료했습니다. :)

**Comment by seokho-son** [comment]
Thanks ! @copyx 
/lgtm

**Pull request merged**